web_accounts_list.json

```diff
@@ -366,6 +366,17 @@
       "category" : "XXXPORNXXX",
       "valid" : true
      },
+     {
+      "name" : "bdsmsingles",
+      "check_uri" : "https://www.bdsmsingles.com/members/{account}/",
+      "account_existence_code" : "200",
+      "account_existence_string" : "<title>Profile",
+      "account_missing_string" : "BDSM Singles",
+      "account_missing_code" : "200",

```

Comment:
For this `account_missing_code` it should be a 302 as there is a redirect to the main page when an invalid user is requested.